x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/forecasts_table/forecasts_table.js

```diff
@@ -163,6 +203,35 @@ export class ForecastsTable extends Component {
     await navigateToUrl(singleMetricViewerForecastLink);
   }
 
+  async deleteForecast(forecastId) {
+    const {
+      services: {
+        mlServices: { mlApi },
+      },
+    } = this.context;
+
+    this.setState({
+      isLoading: true,
+      isConfirmModalVisible: false,
+    });
+
+    try {
+      await mlApi.deleteForecast({ jobId: this.props.job.job_id, forecastId });
+    } catch (error) {
+      this.setState({
+        isLoading: false,
+        errorMessage: i18n.translate(
+          'xpack.ml.jobsList.jobDetails.forecastsTable.deleteForecastErrorMessage',
+          {
+            defaultMessage: 'An error occurred deleting the forecast.',
```

Comment:
Nit.

```suggestion
            defaultMessage: 'An error occurred when deleting the forecast.',
```

Reply:
Updated in d29de18790177bc0c0d512a43845f70518ab6dc7